crates/router/src/connector/rapyd/transformers.rs

```diff
@@ -463,25 +481,37 @@ pub enum RapydWebhookObjectEventType {
     RefundCompleted,
     PaymentRefundRejected,
     PaymentRefundFailed,
-}
-
-impl TryFrom<RapydWebhookObjectEventType> for api::IncomingWebhookEvent {
-    type Error = error_stack::Report<errors::ConnectorError>;
-    fn try_from(value: RapydWebhookObjectEventType) -> Result<Self, Self::Error> {
-        match value {
-            RapydWebhookObjectEventType::PaymentCompleted => Ok(Self::PaymentIntentSuccess),
-            RapydWebhookObjectEventType::PaymentCaptured => Ok(Self::PaymentIntentSuccess),
-            RapydWebhookObjectEventType::PaymentFailed => Ok(Self::PaymentIntentFailure),
-            _ => Err(errors::ConnectorError::WebhookEventTypeNotFound).into_report()?,
-        }
+    PaymentDisputeCreated,
+    PaymentDisputeUpdated,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, strum::Display)]
+#[serde(rename_all = "UPPERCASE")]
+pub enum RapydWebhookDisputeStatus {
+    Act,
+    Rvw,
+    Los,
+    Win,
+}
```

Comment:
Either add documentation comments to explain what these are, or use full names for the variants and use `#[serde(rename = "...")]` like so:

```rust
enum RapydWebhookDisputeStatus {
    #[serde(rename = "RVW")]
    Review,
    // ...
}
```